### PR TITLE
Hide timeframe related values on /pricing

### PR DESCRIPTION
**Pull request by D13ce**
This PR
* Hides all values, where we don't know what we can support yet, by writing "Soon" in italic instead

### CODE REVIEW

File: frontend/components/pricing/PremiumCompare.vue [hunk 1]
```diff
@@ -47,10 +49,11 @@ const rows = computed(() => {
 
     return {
       value,
-      tooltip
+      tooltip,
+      class: undefined as string | undefined
```
Comment (MauserBitfly) [inline frontend/components/pricing/PremiumCompare.vue:53]: that is not pretty, I would suggest to define the return type CompareValue
```
const mapValue = (property: string, perks: PremiumPerks):CompareValue => {
```

then you should be able to remove the whole row here.

**Review by MauserBitfly — COMMENTED** [review]
looks good, just one Nitpick

File: frontend/components/pricing/PremiumCompare.vue (hunk 1 above)
Comment (D13ce) [reply frontend/components/pricing/PremiumCompare.vue]: Obviously better. I couldn't wrap my head around that at first. Thanks!

**Review by MauserBitfly — APPROVED** [review]
LGTM

**Pull request merged**